source/server/hot_restart_impl.cc

```diff
@@ -472,23 +445,25 @@ void HotRestartImpl::terminateParent() {
 void HotRestartImpl::shutdown() { socket_event_.reset(); }
 
 std::string HotRestartImpl::version() {
-  return versionHelper(shmem_.maxStats(), Stats::RawStatData::maxNameLength(), *stats_set_);
```

Comment:
@mrice32 I wanted BlockRawStatDataAllocator's `stats_set_` member variable to be private, so I ended up writing a function here [1] to just fetch the version as a string.

[1] https://github.com/ambuc/envoy/pull/1/files#diff-fb83459430ccfd2f0d934dcfacdbd9d1R399

Reply:
Seems reasonable to me.